config/kube-acp-stack.yaml

```diff
@@ -0,0 +1,68 @@
+hazelcast:
+  enabled: true
+  cluster:
+    memberCount: 1
+  mancenter:
+    enabled: false
+cockroachdb:
+  enabled: true
+  single-node: true
+  statefulset:
+    replicas: 1
+acp:
+  enabled: true
+  serverURL: "https://acp.acp-system:8443"
+  ingress:
+    enabled: true
+    hosts:
+      - host: acp.acp-system
+        paths:
+          - path: /
+  features:
+    integration_endpoints: true
+    swagger_ui: true
+  importJob:
```

Comment:
Should we use acp-cd instead legacy import job? @ppodbielski-cloudentity @mbilski